defaults.go

```diff
@@ -265,10 +240,10 @@ var defaultTemplate = `
                     </tr>
                     <tr>
                       <td>
-                        Time:
+                        Started at:
                       </td>
                       <td>
-                        {{ duration build.started_at build.finished_at }}
+                        {{ datetime build.started Basic Local }}
```

Comment:
After carefully reading the time docs again it appears I misinterpreted the format option. I'll fix it.